Sources/Frisbee/Interactors/BodyBuilder.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+struct BodyBuilder: BodyBuildable {
```

Comment:
Create tests for this struct

Reply:
![bodybuilder-porra](https://user-images.githubusercontent.com/421758/34939687-e39c9470-f9d3-11e7-9bfb-5bf7e979794e.jpg)